<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>475</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="StartWork">
       <property name="text">
        <string>Начать работу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CameraCalculate">
       <property name="text">
        <string>Расчёт высоты камеры и времени</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ImageFromCamera">
       <property name="text">
        <string>Изображение с камеры</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SpeedTest">
       <property name="text">
        <string>Тест скорости распознавания</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;Создано при поддерже&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;фонда содействия иновациям&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/umnik_2.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="im2.qrc">:/newPrefix/umnik_2.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanBut">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlotsOut">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывод графиков</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Recomendations">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Рекомендации</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveReport">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить отчёт</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="CloseWindow"/>
   </widget>
   <widget class="QMenu" name="Info">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="Acknowledgments"/>
    <addaction name="Links"/>
    <addaction name="AboutWork"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="OpenModel">
   <property name="text">
    <string>Окрыть модель нейросети</string>
   </property>
  </action>
  <action name="SaveSettings">
   <property name="text">
    <string>Сохранить настройки</string>
   </property>
  </action>
  <action name="CloseWindow">
   <property name="text">
    <string>Закрыть окно</string>
   </property>
  </action>
  <action name="Acknowledgments">
   <property name="text">
    <string>Благодарности</string>
   </property>
  </action>
  <action name="Links">
   <property name="text">
    <string>Ссылка на исходники</string>
   </property>
  </action>
  <action name="AboutWork">
   <property name="text">
    <string>Описание работы</string>
   </property>
  </action>
  <action name="Language">
   <property name="text">
    <string>Язык</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="im1.qrc"/>
  <include location="im3.qrc"/>
  <include location="im2.qrc"/>
 </resources>
 <connections/>
</ui>
